brightwind/analyse/plot.py

```diff
@@ -89,14 +96,29 @@ def _set_col_map(col_map_colors):
     def color_map(self):
         return self._color_map
 
+    @property
+    def color_map_symmetric(self):
+        return self._color_map_symmetric
+
     @color_map.setter
     def color_map(self, col_map_colors):
         self._color_map = self._set_col_map(col_map_colors)
 
+    @color_map.setter
+    def color_map_symmetric(self, col_map_colors):
```

Comment:
@BiancaMorandi 
I'm not sure this works. The `@color_map.setter` I think is a setter for the `color_map` property. It looks like you have another property `color_map_symmetric` that you want to set so it might need to be `@color_map_symmetric.setter`?

Reply:
solved

Reply:
solved